usecases/cost.go

```diff
@@ -47,35 +46,29 @@ func SetActivitiesCost(serv *components.Service, bodyBytes []byte) (err error) {
 	var jsonData map[string]interface{}
 	err = json.Unmarshal(bodyBytes, &jsonData)
 	if err != nil {
-		log.Printf("Error unmarshalling JSON data: %v", err)
-		return
+		return fmt.Errorf("unmarshalling JSON data: %w", err)
 	}
 	formVersion, ok := jsonData["version"].(string)
 	if !ok {
-		log.Printf("Error: 'version' key not found in JSON data")
-		return
+		return fmt.Errorf("'version' key not found in JSON data")
 	}
 	var acForm forms.ActivityCostForm_v1
 	switch formVersion {
 	case "ActivityCostForm_v1":
 		var f forms.ActivityCostForm_v1
 		err = json.Unmarshal(bodyBytes, &f)
 		if err != nil {
-			log.Println("Unable to extract new activity costs request ")
-			return
+			return fmt.Errorf("unable to extract new activity costs request ")
 		}
 		acForm = f
 	default:
-		err = errors.New("unsupported version of activity costs form")
-		return
+		return fmt.Errorf("unsupported version of activity costs form")
```

Comment:
Might want to include the form version in the error msg, so it's easier to debug.